### PR TITLE
Change some wording

### PR DESCRIPTION
**Pull request by Hjaltesorgenfrei**
Not super happy with the last paragraph

### CODE REVIEW

**Review by tobloef — APPROVED** [review]

File: report/50_Conclusion.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
-While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work. If the techniques discussed in this report were utilized more broadly in the industry, both for web development projects and otherwise, being a developer would likely be a less stressful job. But it is also important for each team to consider what parts will provide the most value, as there is famously no silver bullet and because each DevOps practice can require a significant time effort for it to be implemented properly. This is especially true when considering adding new technology to a project, as buzzwords and hype can often blind developers into grabbing a solution and then go looking for a problem. That is, developers should be careful not to focus too much on tools instead of the overall philosophy of DevOps.
+While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work.
```
Comment (Jakobis) [inline report/50_Conclusion.md:5]: Having this sentence but not expanding on it is pretty weird.

File: report/50_Conclusion.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work. 
+If the techniques discussed in this report were utilized more broadly in the industry, both for web development projects and otherwise, being a developer would likely be a less stressful job. 
+But it is also important for each team to consider what parts will provide the most value, as there is famously no silver bullet and because each DevOps practice can require a significant time effort for it to be implemented properly.
```
Comment (Jakobis) [inline report/50_Conclusion.md:7]: Aw yeah, Paolo is gonna love this

File: report/50_Conclusion.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work. 
+If the techniques discussed in this report were utilized more broadly in the industry, both for web development projects and otherwise, being a developer would likely be a less stressful job. 
+But it is also important for each team to consider what parts will provide the most value, as there is famously no silver bullet and because each DevOps practice can require a significant time effort for it to be implemented properly. 
+This is especially true when considering adding new technology to a project, as buzzwords and hype can often blind developers into grabbing a solution and then go looking for a problem. 
+That is, developers should not focus too much on specific tools, and should instead follow the general philosophy of DevOps.
```
Comment (Jakobis) [inline report/50_Conclusion.md:9]: Perhaps replace "and should instead" with "but rather

**Review by Jakobis — CHANGES_REQUESTED** [review]

File: report/50_Conclusion.md (hunk 1 above)
Comment (tobloef) [reply report/50_Conclusion.md:5]: Hvilken sætning, det er lidt svært at se?

Comment (Jakobis) [reply report/50_Conclusion.md:5]: While working on the project it has become clear that while great benefits can be gained by adopting a DevOps mindset, it is also a lot of work.

**Pull request merged**